sentry_sdk/client.py

```diff
@@ -336,19 +337,27 @@ def capture_event(
         if session:
             self._update_session_from_event(session, event)
 
-        if event_opt.get("type") == "transaction":
-            # Transactions should go to the /envelope/ endpoint.
-            self.transport.capture_envelope(
-                Envelope(
-                    headers={
-                        "event_id": event_opt["event_id"],
-                        "sent_at": format_timestamp(datetime.utcnow()),
-                    },
-                    items=[
-                        Item(payload=PayloadRef(json=event_opt), type="transaction"),
-                    ],
-                )
+        attachments = hint.get("attachments")
+        is_transaction = event_opt.get("type") == "transaction"
+
+        if is_transaction or attachments:
+            # Transactions or events with attachments should go to the
```

Comment:
Should we document somewhere that you need on-premise installation Sentry version >= v20.6.0 for attachments to work?

Reply:
We currently don't have this for any feature. That said, we talked about something like this. Will take this to slack.